src/release/issue-triaging.md

```diff
@@ -0,0 +1,80 @@
+It's okay if not all of these are always considered, treat it as guideline, not a hard check list. It is also not exhaustive.
+
+- The issue should make sense, that is it should present a problem.
+    - For example, if an issue is a question about Rust in general, the issue should be closed and the user redicted to URLO/Discord.
+      You can of course answer the question too :) (but make sure to mention that the user should go to URLO/Discord next time).
+- Add appropriate labels ([Labels](#labels))
+    - Specifically, `T-*` and `C-*` are the most relevant
+- If the issue contains no reproduction but needs one, ask for one and add the `S-needs-repro` label
+- If the issue could benefit from bisecting the regression, add `E-needs-bisection` (or do the bisection yourself)
+- Does this issue require nightly? Add `requires-nightly`.
+- Is the issue a regression? Apply the `regression-untriaged` label (or figure out what regression it is exactly)
+- If you happen to know people who this issue is relevant to, ping them.
+    - For example, write `cc @ThatPerson` if `ThatPerson` has been working a lot on the problematic feature recently
+- Does this issue require incomplete or internal features? Add `requires-{incomplete,internal}-features`.
+
+For applying and removing labels, unprivileged users can use rustbot.
+For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos -needs-triage`.
+
+To see a list of all labels, check out the "labels" page next to the search bar in the issue tracker.
+
+## Further triaging
+
+For issues that have been through the initial triaging step (that is, don't have the `needs-triage` label anymore), there are usually
+still things that can be improved. There are often many more labels that could be applied (using rustbot again if you don't have privileges).
+
+Additionally, old (there is no clear definition of old yet, but something on the order of months) `S-needs-repro` issues can be closed
+if there is no way to make progress without a reproduction. This requires privileges, but if you don't have them, you can just link the issue
+on Zulip (for example in `t-release/triage` or `general`) and someone with privileges can close it for you.
+
+Another useful thing to do is go through `E-needs-mcve` and `E-needs-bisection` issues and creating minimizations or bisecting the issue
+(using [cargo-bisect-rustc](`https://github.com/rust-lang/cargo-bisect-rustc`)). When you provide one, you can also remove the label
+using rustbot (`@rustbot label -E-needs-bisection`).
+
+## Labels
+
+There are many different labels that can be applied to issues.
+
+- `needs-triage`: signals that an issue is new and needs initial triage
+- `T-*`: Specifies the team or teams that this issue is relevant to, for example compiler, types or libs
+- `C-*`: Specifies the category of the label, for example a bug, tracking issue or discussion
```

Comment:
Should we mention here that <kbd>A-diagnostic</kbd> issues typically don't / shouldn't be marked with <kbd>C-bug</kbd> but just with <kbd>D-*</kbd> if applicable?